features/20-standalone-launch.feature

```diff
@@ -31,6 +31,14 @@ Feature: Follows the SMART standalone launch sequence
         When I ask for authorization without the state field
         Then the response code should not be 200
 
+    Scenario: EHR evaluates authorization request - very long "state"
```

Comment:
The screenshot from further down is actually Allscripts failing this test. I don't know if it's the length or the `'` in the state string, but it does actually return results.